### PR TITLE
Redis Caching for /user/{user_id}

### LINKED ISSUE #138: [Task]: Implement caching for  /users/{user_id}
What does this code do?
Edited  the HttpTriggerAPIUserId trigger to  connect to the Redis cache, to a specified port and host.   It will still connect to an SQL database, but after the user does a "GET" request whatever results that the search for the User ID yields, those results will be dumped into a Redis Cache.

Just to test that it is connected to a Redis Cache, I typed out r.get("3") to see if the user first/last name that corresponds to that particular User Id.

Alternatives:
No other alternatives were considered for this task.

Important things to note: Make sure you type pip install redis and pyodbc to make sure they are both imported.  When I tried testing/playing around with the Redis Cache, I came across errors related to them.  Also, as a HEADS UP,  I was not able to fully test the code because I couldn't download azure.functions for Python.

### PR DESCRIPTION
**Pull request by chonger878**
Closes #138 

This PR will be adding code for Redis caching to the HttpTriggerAPIUserId Trigger file, especially when the user uses the "GET" request to search for a user by User ID, and then dumps the search into a Redis Cache.  

I slipped in a line of code- r.get("3") for the user to test if they can connect to the Redis cache, might be not necessary.

I give credit to PR #170,  PR# 173, and PR #184  for guiding me through writing the code for Redis caching.

### CODE REVIEW

**Review by kieranmiguel — APPROVED** [review]

**Comment by kieranmiguel** [comment]
Please update with key password from slack, and hardcoded port and host from Johart's PR

**Review by jaguiniga — APPROVED** [review]
Password is protected by GitHub Secrets. Host URL is luca's Redis Server Resource. You are collecting User/UserId. Checking the SQL DB and dumping in the cache.

File: functions/HttpTriggerAPIUsersId/__init__.py [hunk 1]
```diff
@@ -11,6 +12,13 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
     logging.info(
         'Python HTTP trigger for users/userId is processing a request ')
 
+    #Initiating REDIS cache
+    REDIS_HOST = 'nsc-redis-dev-usw2-thursday.redis.cache.windows.net'
+    r = redis.Redis(host= REDIS_HOST, port= 6380, db= 0, password= '${{ secrets.ENV_REDIS_KEY }}', ssl= True)
+
+    #Something to test the redis cache
+    r.get("2")
```
Comment (toddysm) [inline functions/HttpTriggerAPIUsersId/__init__.py:20]: You should not have hard-coded values. Not sure why this is here?

File: functions/HttpTriggerAPIUsersId/__init__.py [hunk 2]
```diff
@@ -11,6 +12,13 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
     logging.info(
         'Python HTTP trigger for users/userId is processing a request ')
 
+    #Initiating REDIS cache
+    REDIS_HOST = 'nsc-redis-dev-usw2-thursday.redis.cache.windows.net'
```
Comment (toddysm) [inline functions/HttpTriggerAPIUsersId/__init__.py:16]: What if the host changes? This should be pulled from app configuration/environment

**Pull request merged**